parler/tests/utils.py

```diff
@@ -13,14 +13,23 @@
 except ImportError:
     from django.utils.importlib import import_module  # Python 2.6
 
+try:
+    User = get_user_model()
+except ImportError:  # django < 1.5
+    from django.contrib.auth.models import User
+
 
 def clear_cache():
     """
     Clear internal cache of apps loading
     """
+    import django
     if django.VERSION >= (1.7):
-        from django.db.models import loading
-        loading.cache.loaded = False
+        try:
+            from django.db.models import loading
+            loading.cache.loaded = False
+        except ImportError:  # Django >= 1.9
+            pass
```

Comment:
One question about this part: does that mean that in Django 1.9 a cache clear is no longer needed?